core_gems.rb

```diff
@@ -1,12 +1,13 @@
 dir 'core_gems'
 download "bundler", "1.10.6"
-download "json", "1.8.3"
-download "msgpack", "0.5.11"
+download "json", "2.1.0"
+download "msgpack", "1.2.2"
 download "cool.io", "1.4.4"
 download "http_parser.rb", "0.6.0"
-download "yajl-ruby", "1.2.1"
+download "yajl-ruby", "1.3.1"
 download "sigdump", "0.2.4"
 download "thread_safe", "0.3.5"
-download "oj", "2.15.1"
+download "oj", "3.3.10"
 download "tzinfo", "1.2.2"
 download "tzinfo-data", "1.2016.5"
+download "google-protobuf", "3.5.1"
```

Comment:
This seems new. Any reason we're doing this explicitly, rather than letting another gem pull it in as a dependency?

Reply:
I had to explicitly pin `google-protobuf` to `3.5.1` because otherwise it pulls in the latest `3.5.1.1` which is somehow broken.

Reply:
Let's update the PR description, then.

Reply:
Updated. Also opened a ticket to report the issue with `3.5.1.1`: https://github.com/google/protobuf/issues/4210.

Reply:
:+1: